sdk/core/Azure.Core/README.md

```diff
@@ -1,20 +1,20 @@
 # Azure Core shared client library for .NET
 
-Azure.Core provides shared primitives, abstractions, and helpers for modern .NET Azure SDK client libraries. 
-These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) 
-and can be easily identified by package and namespaces names starting with 'Azure', e.g. ```Azure.Storage.Blobs```. 
-A more complete list of client libraries using Azure.Core can be found [here](https://github.com/Azure/azure-sdk-for-net#core-services). 
+Azure.Core provides shared primitives, abstractions, and helpers for modern .NET Azure SDK client libraries.
+These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html)
+and can be easily identified by package and namespaces names starting with 'Azure', e.g. ```Azure.Storage.Blobs```.
```

Comment:
Nit: This works with commonmark but is non-standard for inline code. While you're already changing it,
```suggestion
and can be easily identified by package and namespaces names starting with 'Azure', e.g. `Azure.Storage.Blobs`.
```

Reply:
There's a few other places below, but up to you, of course.